smart_campus/app/templates/app/reward_add_page.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'app/base.html' %} {% load static %} {% block stylesheet %} {{ block.super }}
```

Comment:
dd proper blank lines can enhance readability.
DO NOT remove them for no reason.

Reply:
okay, indented it.